File: exercise/examples/absolute.py
print ("***************************")
print ("Absolute value program...")
print ("To quit program,please press q... ")
print ("***************************")
def absolute (value):
    if (value >= 0):
        return value
    return -value
while (True):
    value =raw_input ("Enter the number:")
    if (value=="q"):
        print ("Exiting program")
        break
    value = int (value)
    print (absolute(value))

File: exercise/examples/absolute.c
#include <stdio.h>
int absolute(int);
int main (void)
{	
	while (1)
	{
		char value ;
		scanf ("%c",&value);
		if (value == 'q')
		{
			printf ("Exiting program ...\n");
			break;
		}
		int temp = (int)value;
		printf ("Temp:%d\n",temp);
		printf ("%d\n",absolute (temp));
		return 0;

	
	}	
}
int absolute (int value)
{
	if (value<0)
	{
		value = -value;
	}
	return value;
}
